chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/search/CertifiedProductSearchDAOImpl.java

```diff
@@ -130,4 +151,47 @@ public CertifiedProduct getByChplProductNumber(String chplProductNumber)
 		}
 		return results;
 	}
+	
+	private IcsFamilyTreeNode convertIcs(CertifiedProductBasicSearchResultEntity result) {
+		IcsFamilyTreeNode node = new IcsFamilyTreeNode();
+		node.setId(result.getId());
+		node.setChplProductNumber(result.getChplProductNumber());
+		node.setCertificationDate(result.getCertificationDate());
+		CertificationStatus cs = new CertificationStatus(result.getCertificationStatus());
+		node.setCertificationStatus(cs);
+		Developer dev = new Developer();
+		dev.setName(result.getDeveloper());
+		node.setDeveloper(dev);
+		Product prod = new Product();
+		prod.setName(result.getProduct());
+		node.setProduct(prod);
+		ProductVersion pv = new ProductVersion();
+		pv.setVersion(result.getVersion());
+		node.setVersion(pv);
+		ArrayList<CertifiedProduct> childrenList = new ArrayList<CertifiedProduct>();
+		if(result.getChild() != null){
+			String[] children = result.getChild().split("☺");
+			for(String child : children){
+				String[] childInfo = child.split("☹");
```

Comment:
Is it possible these characters are set somewhere in properties files or something? If they are, you should use them here.

Reply:
I cannot find anywhere in the code where they are currently properties, but I can add them as properties if they should be.

Reply:
If they're not already there's no need. If, in the future, we find ourselves needing to change them then we can extract it